app/controllers/PreferencesProxy.scala

```diff
@@ -0,0 +1,40 @@
+package controllers
+
+import com.gu.pandomainauth.PanDomainAuthSettingsRefresher
+import com.gu.workflow.util.{Code, Dev}
+import config.Config
+import play.api.Logging
+import play.api.libs.ws.WSClient
+import play.api.mvc.{BaseController, ControllerComponents}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+/* Proxy for editorial-preferences, to help with latency since workflow-frontend sits behind CloudFront*/
+class PreferencesProxy(
+  override val config: Config,
+  override val controllerComponents: ControllerComponents,
+  override val wsClient: WSClient,
+  override val panDomainSettings: PanDomainAuthSettingsRefresher
+) extends BaseController with PanDomainAuthActions with Logging {
+
+  private def proxyRequest(relativePath: String) = APIAuthAction.async { request =>
+    val url = s"${config.preferencesUrl}/$relativePath"
+    wsClient.url(url)
+      .withHttpHeaders(request.headers.toSimpleMap.toSeq: _*)
+      .execute(request.method)
+      .map { response =>
+        Ok(response.body).withHeaders(response.headers.mapValues(_.head).toSeq: _*)
```

Comment:
I think you should also handle passing on the HTTP status code to the client

Reply:
absolutely - done in https://github.com/guardian/workflow-frontend/pull/408/commits/85689dfa9ae4ddbb2a223a35ca4a27c604d675f9